### PR TITLE
adds greeting and temporary description to home page, basic styling, …

### PR DESCRIPTION
**Pull request by marilynremigio**
…and passes test

### CODE REVIEW

File: app/javascript/components/pages/Home.js [hunk 1]
```diff
@@ -2,7 +2,36 @@ import React from 'react'
 
 const Home = () => {
   return (
-    <div>Home</div>
+    <>
```
Comment (sjproctor) [inline app/javascript/components/pages/Home.js:5]: Just a note - you don't need a fragment and a div. As long as one thing is wrapping all other tags.

File: app/javascript/components/pages/Home.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { BrowserRouter } from 'react-router-dom'
+
+describe("<Home />", () => {
+    it("renders an header", () => {
+      render(
+        <BrowserRouter>
+          <Home />
+        </BrowserRouter>
+      )
+    //   screen.logTestingPlaygroundURL()
```
Comment (sjproctor) [inline app/javascript/components/pages/Home.test.js]: This can be removed. ^^

**Review by sjproctor — APPROVED** [review]
Looks goos, just a few comments for your consideration.

File: app/javascript/components/pages/Home.test.js (hunk 2 above)
Comment (marilynremigio) [reply app/javascript/components/pages/Home.test.js]: Gotcha

**Pull request merged**